docs-new/mkdocs.yml

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+site_name: Apache Iceberg
+docs_dir: home
+
+theme:
+  name: material
+  language: en
+  logo: assets/images/iceberg-logo-icon.png
+  favicon: assets/images/favicon-96x96.png
+  features:
+    - navigation.tabs
+    - navigation.path
+    - navigation.top
+    - navigation.tracking
+    - toc.follow
+    - search.suggest
+    - search.highlight
+    - content.tabs.link
+    - content.code.copy
+    - content.code.annotate
+
+plugins:
+  - search
+  - macros:
+      include_yaml:
+        - variables.yml
+  - monorepo
+
+nav:
+  - Quickstart:
+    - Hive: hive-quickstart.md
+    - Spark: spark-quickstart.md
+  - Docs: 
+    - latest: '!include home/docs/latest/mkdocs.yml'
+    - 1.3.1: '!include home/docs/1.3.1/mkdocs.yml'
+#    - 1.3.0: '!include home/docs/1.3.0/mkdocs.yml'
+  - Releases: releases.md
+  - Roadmap: roadmap.md
+  - Blogs: blogs.md
+  - Talks: talks.md
+  - Vendors: vendors.md
+  - Project: 
+    - Join: community.md
+    - Spec: spec.md
+    - View spec: view-spec.md
+    - Puffin spec: puffin-spec.md
+    - Multi-engine support: multi-engine-support.md
+    - How to release: how-to-release.md
+    - Terms: terms.md
+  - ASF:
+    - ASF: ASF.md
```

Comment:
I don't think we need this page. It doesn't have any content and wasn't in the [old mkdocs site](https://github.com/apache/iceberg/blob/apache-iceberg-0.13.2/site/mkdocs.yml).

Reply:
Yeah, I'll play with this. I forgot I had this here and mkdocs was doing something funky with the lack of any index md file

Reply:
So ultimately there's an issue with how mkdocs renders.

https://github.com/squidfunk/mkdocs-material/issues/868

The solution is to provide an [override](https://squidfunk.github.io/mkdocs-material/customization/#overriding-partials) to the materials theme. I'm going to dig into that later though. I've removed the ASF.md file for now. The current functionality is that the "ASF" nav link just links directly to "Sponsorship" (the first sub nav item in the index) so for now I'm just linking to community.md so that we'll at least be able to open a local page to see the nav.